rhtap-cli/rhtap-local-install.sh

```diff
@@ -94,8 +94,15 @@ run-rhtap-e2e() {
   export APPLICATION_ROOT_NAMESPACE="rhtap-app"
   export GITHUB_ORGANIZATION="<REPLACE_ME>"
   export GITLAB_ORGANIZATION="<REPLACE_ME>"
-  export QUAY_IMAGE_ORG="<REPLACE_ME>"
-  export IMAGE_REGISTRY="<REPLACE_ME>"  ## "quay.io" or "$(kubectl -n rhtap-quay get route rhtap-quay-quay -o 'jsonpath={.spec.host}')"
+  export QUAY_IMAGE_ORG
+  export IMAGE_REGISTRY
+  if [ "$quay_install_enabled" = "false" ]; then
+    IMAGE_REGISTRY="quay.io"
+    QUAY_IMAGE_ORG="<REPLACE_ME>"
+  else
+    IMAGE_REGISTRY="$(kubectl -n rhtap-quay get route rhtap-quay-quay -o 'jsonpath={.spec.host}')"
+    QUAY_IMAGE_ORG=rhtap
```

Comment:
I would move quay image org var at the beginning of the file with default value "rhtap", but with note # if you are using different image org than rhtap, then change this env var too (or something similar). I think everyone is using their own image org.

Reply:
Ok, then  I don't set any default values for them , for example. They can use any Quay server and Image org. 
```
  export QUAY_IMAGE_ORG="<REPLACE_ME>"
  export IMAGE_REGISTRY="<REPLACE_ME>"
```

My initial thought is that because we often use either quay.io or the Quay server installed along with RHTAP. I suppose if we are using quay.io, then each of us will use our own image org, if we are using the  Quay server installed along with RHTAP, then we can use the default image org `rhtap` and  the quay host with  command`$(kubectl -n rhtap-quay get route rhtap-quay-quay -o 'jsonpath={.spec.host}')`

Reply:
WDYT?

Reply:
fixed